deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/normalization/BatchNormalization.java

```diff
@@ -390,10 +390,10 @@ public INDArray preOutput(INDArray x, TrainingMode training, LayerWorkspaceMgr w
         int dim = 1;
         if(x.rank() == 4 && layerConf().getCnn2DFormat() == CNN2DFormat.NHWC)
             dim = 3;
-        if(x.size(dim) != layerConf().getNOut()){
+  /*      if(x.size(dim) != layerConf().getNOut()) {
             throw new IllegalArgumentException("input.size(" + dim + ") does not match expected input size of " + layerConf().getNIn()
                     + " - got input array with shape " + Arrays.toString(x.shape()));
-        }
+        }*/
```

Comment:
This looks like it is temporary, did you forget to revert that? If not remove it outright instead of commenting it out.

Reply:
@treo yeah that's why it's not marked for review yet :)